### PR TITLE
kill -9 if fail to stop

### PR DESCRIPTION
**Pull request by markelliot**

### CODE REVIEW

File: gradle-sls-packaging/src/main/resources/init.sh [hunk 1]
```diff
@@ -99,7 +99,7 @@ stop)
         PID=$(cat $PIDFILE)
         kill $PID
         COUNTER=0
-        while is_process_service $PID $SERVICE && [ "$COUNTER" -lt "240" ]; do
+        while is_process_service $PID $SERVICE && [ "$COUNTER" -lt "180" ]; do
```
Comment (iamdanfox) [inline gradle-sls-packaging/src/main/resources/init.sh]: We've arbitrarily changed the shutdown timeouts in a variety of projects recently, and it's pretty hard to keep track of what the actual source of truth is.

Please could we just pick something a number and put it in the sls-spec, and then make everyone comply with this?  (I think some places might have lower timeouts than 3 minutes)

Comment (markelliot) [reply gradle-sls-packaging/src/main/resources/init.sh]: I'm happy to leave this part alone and just add the kill -9 after the timeout. for a long time, the behavior of this project actually defined the number.

there's a discussion on the spec now, but IMO the spec should define the API, and the timeout here seems like an implementation detail to me.

**Comment by jboreiko** [comment]
Magritte would like to see this go in.  Some of the bootvisor's struggles appear to come from failing to stop processes.  It's not really an option to allow the managed process to continue and it forces us to have to access the box to kill them.

The timeout still strikes me as too long given jetty's default graceful shutdown in 30 seconds.

**Comment by jboreiko** [comment]
I would prefer something closer to https://github.com/palantir/sls-packaging/pull/305 for the timeout.

**Comment by markelliot** [comment]
I've removed the timeout change so that can be considered separately

**Review by iamdanfox — APPROVED** [review]

**Comment by iamdanfox** [comment]
cc @gregakinman - I guess we're gonna need to also add this functionality in go-init right?

**Comment by gregakinman** [comment]
yeah thanks for the heads up

**Comment by iamdanfox** [comment]
Confirming this `kill -9` functionality was also implemented in go-init: https://github.com/palantir/go-java-launcher/pull/49/files#diff-fdcff91f81dff786b4df5f55c08f423dR97

**Pull request merged**